all_repos/push/azure_repos_pull_request.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+import base64
+import json
+import subprocess
+from typing import NamedTuple
+
+from all_repos import autofix_lib
+from all_repos import azure_repos_api
+from all_repos import git
+from all_repos.util import hide_api_key_repr
+
+
+class Settings(NamedTuple):
+    api_key: str
+    organization: str
+    project: str
+    base_url: str = 'https://dev.azure.com'
+    api_version: str = '6.0'
```

Comment:
is this actually something that should be configurable?  I would imagine there would need to be _code_ changes to use a different api version and is not something that should be user-changeable